functions-and-scope.js

```diff
@@ -73,6 +101,19 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // ---- Verwachte uitkomst: 9
 
 
+
+function highestNumber (gradeArray) {               //3a en 3b door parameter gradeArray mee te geven kun je iedere array in deze functie kwijt
+    let highestGrade = 0
```

Comment:
Het is geen goed voorbeeld, maar ik wil het je wel meegeven:
Wanneer je een array meekrijgt ([-1, -3, -2]) met alleen maar negatieve getallen, dan zal de functie altijd 0 teruggeven. Het is daarom verstandig om altijd de eerste waarde in de array te pakken:
```suggestion
    let highestGrade = gradeArray[0];
```